### PR TITLE
fix: prettier config

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
build 

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
NA

**Summary**
Updates prettier config to use sensible defaults

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

File: packages/generators/src/handlers/default.ts [hunk 1]
```diff
@@ -15,104 +15,68 @@ const resolveFile = (file: string): string => {
...
-        self,
-        'extractPlugin',
-        'Do you want to extract CSS for every file?',
-        ['No', 'Only for Production', 'Yes'],
-        'No',
-        self.force,
-    );
+    const { isPostCSS } = await Question.Confirm(self, "isPostCSS", "Will you be using PostCSS in your project?", cssType == "CSS only", self.force);
+
+    const { extractPlugin } = await Question.List(self, "extractPlugin", "Do you want to extract CSS for every file?", ["No", "Only for Production", "Yes"], "No", self.force);
```
Comment (snitin315) [inline packages/generators/src/handlers/default.ts:56]: This was more readable before.

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts:56]: Weird, it should be breakable https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAExNwB4wBOAhmDAAoA2ArgOYCWUmAvpgLybEDux9WAitTgZ60AHQAZehgAUaOJQBmAGkwAdEPiKkKNBlA2qNAEQiYAnhGqYesTDDNaSZTAGEAyu8yKIhXADc4QnNveko4AH5DTABtDQA5CGiNAHkoShCfP3JCCAATajJRAxAjEABNYQ0AXTLE6PklMSywOABKAG5SkAgABxhitGRQYkJcrnJRhCGUYkoecyHlEAAjZwBrOBh3XtJGWmQiIWW4AFsVuDy8y4liKFpqYlo4ADFfU+IYAfvkEGJqBzdAAWMFOlAA6kC+MJdq13NM+PR-HxzL8wGgliBGPJCBQSLQPshFHN5MsAFZoPAAIQ2W3cxFOcCkUDgRJJcHJlPc+3Cggg8DZlFJIF2hBxvxWxAulG6vUIjBg4PoeRgQOQAA4AAzLOUQeTgki9X5y4RBQLdACO1H5cByfRmfzQAFoWZdLt1CHArfRPTknoSkMShRyQPJTvRDoRjqGeXA+QLA+zljApUqVWqkAAmZMkML7VwQU4BzRoACs3Wo8gAKlKZkHhf4hABJKDXWDuMDy-oAQVb7hg5nCgvkzGYQA

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts:56]: yep it's only affecting typescript, maybe some eslint issue?

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts:56]: Maybe, we need check it

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts:56]: yep it's eslint, setting max width in there too

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts:56]: weird, i.e. plugin don't read pretteir configuration?

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: oh it doesn't pick default prettier rules but when we specify values it works, no need update eslint

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: I've set it to 100, I think 100 makes sense

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: 80 is a bit too small considering types in params as well

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts]: I think small is better more readable, reading all in one line is not good

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: looks good in 100

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts]: Can you provide line?

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: lmk if something is messed up, 80 is a bit too small I feel which leads to over splitting code

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts]: Maybe, but I want to look on examples :smile:

File: packages/generators/src/handlers/default.ts [hunk 2]
```diff
@@ -57,65 +71,83 @@ export async function questions(self: CustomGenerator, Question: Record<string,
...
+            isPostCSS: false,
+            extractPlugin: "No",
+        };
         return;
     }
 
     const { isCSS } =
-        cssType != 'CSS only'
-            ? await Question.Confirm(self, 'isCSS', `Will you be using CSS styles along with ${cssType} in your project?`, true, self.force)
+        cssType != "CSS only"
```
Comment (anshumanv) [inline packages/generators/src/handlers/default.ts:93]: @alexander-akait seems okay to me

Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts:93]: Do you mean before or after?

Comment (anshumanv) [reply packages/generators/src/handlers/default.ts:93]: after

File: packages/generators/src/handlers/default.ts (hunk 1 above)
Comment (anshumanv) [reply packages/generators/src/handlers/default.ts]: yep, @snitin315 are you able to find anything else which looks odd?

File: packages/generators/src/handlers/default.ts (hunk 2 above)
Comment (alexander-akait) [reply packages/generators/src/handlers/default.ts:93]: Yep, after is more readable

**Review by evenstensberg — APPROVED** [review]
mostly looks okay for me.

**Review by snitin315 — APPROVED** [review]

**Pull request merged**